File: leetcode/1051. Height Checker.cpp
class Solution {
public:
    
    
    int heightChecker(vector<int>& heights) {
        int count = 0;
        int n = heights.size();
        int array[n];
        for(int i = 0 ; i < n ; i++){
            array[i] = heights[i];
        }
        sort(heights.begin(),heights.end());
        
        for(int i = 0 ; i < n ; i++){
            if(array[i] != heights[i]){
                count++;
            }
        }
       
        return count;
    }
};

File: online-training-platform/leetcode/1051. Height Checker.kt
class Solution {
    // Time: O(nLogn)
    // Space: O(n)
    fun heightChecker(heights: IntArray): Int {
        val sorted = heights.sorted()
        var result = 0
        for(i in 0 until heights.size){
            if(heights[i] != sorted[i]){
                result += 1
            }
        }

        return result
    }
}